spanner/integration_test.go

```diff
@@ -1569,23 +1569,21 @@ func TestIntegration_BasicTypes(t *testing.T) {
 
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
 	defer cancel()
-	stmts := singerDBStatements
```

Comment:
GitHub renders this in an almost unreadable way, but what it does is it removes the `if !isEmulatorEnvSet()` block and always uses the same data model for both the emulator and real Spanner. The if-block was previously needed as the emulator did not support `JSON`.

Reply:
ack, I made this exact change in my repo last night so I know what u mean here.. thank u for clarifying